roles/ceph-common/tasks/checks/check_mandatory_vars.yml

```diff
@@ -22,6 +22,14 @@
   tags:
     - package-install
 
+- name: make sure ceph_stable_release is set
+  fail:
+    msg: "make sure ceph_stable_release is set to a release name, http://docs.ceph.com/docs/jewel/releases/"
+  when:
+    - ceph_stable_release == 'dummy'
```

Comment:
It's easier for the check to have it defined and also for the users so they know the variable exists and where to find/edit it.